.rubocop.yml

```diff
@@ -786,6 +786,9 @@ Performance/StartWith:
 Performance/StringIdentifierArgument:
   Enabled: true
 
+Performance/StringInclude:
```

Comment:
Most of the changes are due to this rule. It is not strictly backwards compatible since symbols and nil would work for the old format of `%r{/pattern/}.match?(nil)` or `%r{/pattern/}.match?(:pattern)`. It didn't look like our usages would break due to this though (and we probably want to skip the match anyway in the nil case).